### PR TITLE
Enhancement/566 extended bookmark

### PR DESCRIPTION
**Pull request by DnlLrssn**
**Checklist**

- [ ] tests added
- [x] commits conform to the [Commit message format](./CONTRIBUTING.md#commit)
- [x] documentation updated

**Squash commit message**

* 2 new settings to createbookmark action
  - nosheet: Don't include sheet in the bookmark.
  - savelayout: Save layout to bookmark.

* applybookmark changes
  - action no longer logs warning if sheet is missing
  - logs if it has sheet and sheet ID to details of result.

### CODE REVIEW

File: scenario/createbookmark.go [hunk 1]
```diff
@@ -15,9 +16,11 @@ import (
 type (
 	//CreateBookmarkSettings create bookmark settings
 	CreateBookmarkSettings struct {
-		Title       string `json:"title" displayname:"Bookmark title" doc-key:"createbookmark.title"`
-		Description string `json:"description" displayname:"Bookmark description" doc-key:"createbookmark.description"`
-		ID          string `json:"id" displayname:"Bookmark ID" doc-key:"createbookmark.id"`
+		Title           string `json:"title" displayname:"Bookmark title" doc-key:"createbookmark.title"`
+		Description     string `json:"description" displayname:"Bookmark description" doc-key:"createbookmark.description"`
+		ID              string `json:"id" displayname:"Bookmark ID" doc-key:"createbookmark.id"`
+		NoSheetLocation bool   `json:"nosheet" displayname:"Save sheet location" doc-key:"createbookmark.nosheet"`
```
Comment (SeBBBe) [inline scenario/createbookmark.go:22]: Display name says the opposite of what it does - "Save sheet location"

File: scenario/createbookmark.go [hunk 2]
```diff
@@ -15,9 +16,11 @@ import (
...
+		SaveLayout      bool   `json:"savelayout" displayname:"Save Layout" doc-key:"createbookmark.savelayout"`
```
Comment (SeBBBe) [inline scenario/createbookmark.go:23]: L should be lower-case: "Save layout"

File: scenario/createbookmark.go (hunk 1 above)
Comment (DnlLrssn) [reply scenario/createbookmark.go]: oopps.. forgot to change when I changed parameter...

File: generatedocs/data/params.json [hunk 3]
```diff
@@ -231,6 +231,12 @@
     "createbookmark.id": [
         "(optional) ID to use with subsequent `applybookmark` or `deletebookmark` actions. **Note:** This ID is only used within the scenario."
     ],
+    "createbookmark.nosheet": [
+        "Do not include sheet with bookmark."
```
Comment (carlcervin) [inline generatedocs/data/params.json:235]: Maybe change to "Do not include the sheet location in the bookmark."?

File: generatedocs/data/params.json [hunk 4]
```diff
@@ -231,6 +231,12 @@
...
+    ],
+    "createbookmark.savelayout": [
+        "Include layout with bookmark."
```
Comment (carlcervin) [inline generatedocs/data/params.json:238]: Maybe change to "Include the layout in the bookmark."?

**Pull request merged**